packages/frontapp/src/app/components/NotFoundWindow/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FC } from 'react';
+import styled from '@emotion/styled';
+import { colors, dimensions } from '@mimir/ui-kit';
+
+const NotFoundWindowWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: ${colors.bg_secondary};
+  margin: 30px 0px 30px 0px;
+  max-width: 1015px;
+  width: 100%;
+  height: 448px;
```

Comment:
You shouldn't set height here. It should be determined by content height plus paddings